### ISSUE TITLE
Update to React 16

### ISSUE THREAD

**Issue by felixfbecker** [joshwnj/react-visibility-sensor#93]
This warning gets printed when you install react-visibility-sensor in a project that uses React 16:
```
npm WARN react-visibility-sensor@3.10.1 requires a peer of react@^0.14.0 || ^15.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN react-visibility-sensor@3.10.1 requires a peer of react-dom@^0.14.0 || ^15.0.0 but none is installed. You must install peer dependencies yourself.
```
It seems to work fine though

**Comment by r-tock** [comment]
+1

**Comment by joshwnj** [comment]
thanks @felixfbecker & @stevewillard .

published to v3.11.0
